jetty-maven-plugin/pom.xml

```diff
@@ -13,6 +13,7 @@
     <mavenVersion>3.5.0</mavenVersion>
     <pluginToolsVersion>3.5</pluginToolsVersion>
     <bundle-symbolic-name>${project.groupId}.maven.plugin</bundle-symbolic-name>
+    <surefireVersion>2.20.1</surefireVersion>
```

Comment:
Should this be in a management or property section in the top level pom?

Reply:
nope because of https://github.com/eclipse/jetty.project/issues/2128
we will have to wait next surefire release for a general upgrade